### DISCUSSION TITLE
Stuck with react 17

### DISCUSSION THREAD

**Discussion by stevebeauge** [pmndrs/zustand#2859] [Q&A]
Hi,
I'm stuck with react 17 because of some external requirements of a 3rd party app.
Zustand 5 made react 18 the minimal version.
Is there anything in react 18 that justify the move, and is there any chance to revert the requirement? 

(I'll stick on v4 if not, not an actual big deal)

thanks for feedback

**Comment by dai-shi** [comment]
I think it makes sense to stick on Zustand v4 with React 17. Zustand v5 depends on React 18's function.
